lib/matplotlib/testing/exceptions.py

```diff
@@ -1,15 +1,3 @@
-class KnownFailureTest(Exception):
```

Comment:
Numpy's tools are useful for testing using numpy arrays, so I don't find referencing them to be a compelling argument. The exceptions removed here are not unique to matplotlib functionality. It was a bad idea to for people to use them from us in the first place, and are trivially recreated for anyone down stream who happen to use them.